### PR TITLE
Added mlflow page and updated module doc

### PR DESCRIPTION
**Pull request by ketancmaheshwari**
Added a page dedicated to MLflow and updated information about how to use swift and parsl modules.

### CODE REVIEW

File: software/workflows/mlflow.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
+
+*****
+MLflow
+*****
+
+
+Overview
+========
+
+MLflow MLflow is an open source platform to manage the ML lifecycle, including
```
Comment (verolero86) [inline software/workflows/mlflow.rst:10]: Remove duplicate "MLflow MLflow"

**Review by verolero86 — CHANGES_REQUESTED** [review]
Thanks @ketancmaheshwari - I found just one edit but other than that it looks good to me.

File: software/workflows/mlflow.rst (hunk 1 above)
Comment (ketancmaheshwari) [reply software/workflows/mlflow.rst]: Fixed. Thanks!

**Comment by ketancmaheshwari** [comment]
Added a page for Fireworks.

**Review by verolero86 — APPROVED** [review]
Looks good to me! Thank you for updating MLflow section too 🙂

**Pull request merged**